src/audio/codec_adapter/codec_adapter.c

```diff
@@ -230,6 +230,90 @@ static int codec_adapter_params(struct comp_dev *dev,
 	return 0;
 }
 
+static void codec_adapter_copy_to_lib(const struct audio_stream *source,
+				      void *lib_buff, size_t bytes)
+{
```

Comment:
These two functions can be made generic and moved to audio_stream.h. See: https://github.com/thesofproject/sof/pull/3436/commits/f50fed72fbc8570a032945f75122efcb66297026

Reply:
@mrajwa is it OK with you to move this functions to audio_stream.h and rename the functions.
These functions should be generic and do not have anything to do with the current patch:

``` codec_adapter_copy_to_lib -> audio_stream_copy_to_buf```

Reply:
@dbaluta @mrajwa any update here ?